config/locales/en.yml

```diff
@@ -60,6 +60,8 @@ en:
       there_is_no_tags: "There is no tag"
       next_page: "Next page"
       previous_page: "Previous page"
+    show:
+      articles_tagged_as: "Articles tagged as"
```

Comment:
You don't need the quotation marks unless the string includes a YML control character `:` or similar, or markup.